parlai/agents/tfidf_retriever/tfidf_retriever.py

```diff
@@ -81,12 +81,6 @@ def add_cmdline_args(
             type=int,
             help='How many docs to retrieve.',
         )
-        parser.add_argument(
-            '--retriever-mode',
```

Comment:
why are we deleting this key?

Reply:
It's a repeat of the arg on line 96 (90 in the new file)

Reply:
good catch!